app/com/linkedin/drelephant/spark/fetchers/statusapiv1/statusapiv1.scala

```diff
@@ -292,7 +293,8 @@ class ExecutorSummaryImpl(
   var totalShuffleRead: Long,
   var totalShuffleWrite: Long,
   var maxMemory: Long,
-  var executorLogs: Map[String, String]) extends ExecutorSummary
+  var executorLogs: Map[String, String],
+  var peakUnifiedMemory: Map[String, Long]) extends ExecutorSummary
```

Comment:
How do you get peakUnifiedMemroy value? I don't think Spark history server reports it.